flink-runtime/src/main/java/org/apache/flink/runtime/operators/util/BitSet.java

```diff
@@ -83,6 +83,13 @@ public int bitSize() {
         return bitLength;
     }
 
+    public byte[] getData() {
```

Comment:
Maybe `getData` -> `toBytes` or `toByteArray`, and add doc to indicate that currently only supports heap.